### PR TITLE
Profile: Added

### PR DESCRIPTION
**Pull request by m26unkwn**
1. Profile added
2. CSS added
3.

### CODE REVIEW

**Review by dev-ashishsethi — APPROVED** [review]
theres only html css looks good

File: src/components/Cards/like/card.css [hunk 1]
```diff
@@ -17,15 +17,18 @@
   object-fit: cover;
 }
 
-.all-card-title > a {
+.all-card-title {
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
-  text-decoration: none;
```
Comment (Dev-AkshatGupta) [inline src/components/Cards/like/card.css:24]: Suggestion- Keep consisting spacing in between code line for better indentation

**Review by Dev-AkshatGupta — APPROVED** [review]
Just a suggestion is given .
Everything else is cool you are good to merge

File: src/screens/profile/Profile.jsx [hunk 2]
```diff
@@ -1,7 +1,47 @@
...
+              <div className='flex flex-gap jc-between'>
+                <h4>Email Address : </h4>
+                <h4>{userDetails.email}</h4>
+              </div>
+            </div>
+            <button
+              style={{ marginTop: "2rem" }}
+              onClick={(e) => logoutUser(e, navigate)}
+              className='btn outline-primary'>
+              Logout
```
Comment (SinchanaMS) [inline src/screens/profile/Profile.jsx:40]: The logout doesn't work. Please check. 
The rest of the code is good.

**Review by SinchanaMS — APPROVED** [review]
The UI looks neat! Good work!

File: src/screens/profile/Profile.jsx (hunk 2 above)
Comment (m26unkwn) [reply src/screens/profile/Profile.jsx:40]: lemme check

Comment (m26unkwn) [reply src/screens/profile/Profile.jsx:40]: done

File: src/context/reducer/video-reducer.js [hunk 3]
```diff
@@ -22,6 +22,15 @@ export const videoReducer = (state, action) => {
         return playlist;
       });
       return { ...state, playlists: filterPlaylist };
+    case "CLEAR_ALL_DATA_FROM_STATE":
```
Comment (tarunsankhla) [inline src/context/reducer/video-reducer.js:25]: Create constant for variable reusable

**Review by tarunsankhla — APPROVED** [review]
Minor changes

**Pull request merged**